dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/ProcessUtils.java

```diff
@@ -318,7 +308,7 @@ public static void cancelApplication(List<String> appIds, Logger logger, String
 
         Runtime.getRuntime().exec(runCmd);
       } catch (Exception e) {
-        logger.error("kill application error", e);
+        logger.error("kill application error : {}",e.getMessage(),e);
```

Comment:
There is no necessary to change this.
If we use
```
logger.error("kill application error", e);
```
actually, it is the function of [1].
If we use 
```
logger.error("kill application error : {}",e.getMessage(),e);
```
it will be the function[2]. So we do not need to change this.
[1] ![image](https://user-images.githubusercontent.com/29545877/89123415-b7968500-d501-11ea-849e-c58201936ac3.png)
[2] ![image](https://user-images.githubusercontent.com/29545877/89123428-dd238e80-d501-11ea-88b7-656ba432af42.png)